src/Mobs/Spider.cpp

```diff
@@ -43,9 +43,15 @@ void cSpider::EventSeePlayer(cEntity * a_Entity)
 		return;
```

Comment:
The simplest solution is bailing out without pushing anything to the queue. But is there a guarantee that the chunk will eventually light up at some point?